src/NDItk/multigroup/FluxWeights.hpp

```diff
@@ -35,11 +35,6 @@ class FluxWeights : protected base::RealListRecord {
    */
   std::size_t numberGroups() const { return this->size(); }
 
-  /**
-   *  @brief Return the flux weights
-   */
-  auto weights() const { return this->values(); }
```

Comment:
Double checking: we're getting rid of all things like this because now we call values directly?

Reply:
Yes, as long as the record does not have subrecords or if the values in the array have special meanings (like the fission q values).